sdk/textanalytics/Azure.AI.TextAnalytics/perf/Azure.AI.TextAnalytics.Perf/Scenarios/DetectLanguagePerfTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Test.Perf;
+
+namespace Azure.AI.TextAnalytics.Perf
+{
+    public sealed class DetectLanguagePerfTest
+        : TextAnalyticsTest<PerfOptions>
+    {
+        private readonly TextAnalyticsClient _client;
+
+        private List<string> _batchDocuments;
+
+        public DetectLanguagePerfTest(PerfOptions options) : base(options)
+        {
+            // create client
+            _client = new TextAnalyticsClient(new Uri(TestEnvironment.Endpoint), new AzureKeyCredential(TestEnvironment.ApiKey));
+
+            // create input docs
+            var documentSize = 1000; // PR comment -> should be  'options.something()'
```

Comment:
Take a look at [CountOptions](https://adpnetsource.azurewebsites.net/#Azure.Test.Perf/CountOptions.cs) for a built-in way to support a `Count` option.